src/rf/js/src/home/components/modals.js

```diff
@@ -293,8 +327,12 @@ var UploadModal = React.createBackboneClass({
             }
         }
 
-        if (!layerData.area) {
-            errors.area = ['Please enter an area.'];
+        if (layerData.capture_start && !layerData.capture_end) {
+            errors.capture_end.push('Please enter a date.');
+        }
+
+        if (layerData.capture_end && !layerData.capture_start) {
+            errors.capture_start.push('Please enter a date.');
         }
```

Comment:
Maybe I don't understand what you're asking, but the idea here is that if the user enters one date, then they should enter another. However, entering no dates is okay.